srcpkgs/otfcc/template

```diff
@@ -12,7 +12,7 @@ maintainer="Renato Aguiar <renato@renag.me>"
 license="Apache-2.0"
 homepage="https://github.com/caryll/otfcc"
 distfiles="https://github.com/caryll/otfcc/archive/v${_version}.tar.gz"
-checksum=3604d69e4b427c0e22e3a01acaaccc9b2ccd3df42bdcf52263fe201ce921463c
+checksum=81f8b9b8108c415a645e7e8a407a028531346e61fb570fec541f4aaa4703b1e3
 
 # Yes, there are architectures besides x86 and x64 :-P
 case "$XBPS_TARGET_MACHINE" in
```

Comment:
can't i686* be joined into the arm*|mips* and x86_64* and aarch64* be joined as well ?